src/burn/engine/variable.cpp

```diff
@@ -1897,26 +1897,35 @@ static HRESULT InitializeVariableSystemFolder(
     BOOL f64 = (BOOL)dwpData;
     WCHAR wzSystemFolder[MAX_PATH] = { };
 
-#ifndef _WIN64
```

Comment:
This code used to return the right answer when running on a 32-bit OS, and swap SystemFolder with System64Folder on a 64-bit OS. Now it returns the right answer on 32-bit and 64-bit. Theoretically, someone could have been assuming that they're always deploying to a 64-bit OS (because their org's IT takes care of that) and then took a dependency on them being swapped. Are we justifying this breaking change by preferring the ability to support both 32-bit and 64-bit OS's? Should we have a warning somewhere?